src/sctools/platform.py

```diff
@@ -6,45 +6,40 @@ class PlatformBase:
     """Abstract base class for platform specific metadata"""
 
     @classmethod
-    def get_tags(cls, sequencing_read):
+    def attach_barcodes(cls, args=None):
+        """command line entrypoint for attaching barcodes to a bamfile."""
         raise NotImplementedError
 
+
+class TenXV2(PlatformBase):
+
+    cell_barcode = fastq.EmbeddedBarcode(start=0, end=16, quality_tag='CY', sequence_tag='CR')
+    molecule_barcode = fastq.EmbeddedBarcode(start=16, end=26, quality_tag='UY', sequence_tag='UR')
```

Comment:
Where do these numbers come from? Also what do these tag abbreviations `CY`, `UY`, etc. stand for?

Reply:
These are 10x v2 chemistry definitions. The tags are (soon to be) GA4GH standard tags for cell and molecular barcodes and their qualities. Would it be helpful to add links? Comments?

Reply:
I think comments would be helpful for clarifying what the tags are for the barcodes. For the start and end numbers, you could put those into variables that describe what they are, e.g. `CELL_BARCODE_END= 16` (or whatever name makes sense to you)

Reply:
I think the fact that it's an embedded barcode object with a start and end is maybe enough here, instead of explicitly expanding out into another 12 variables. 

That said, I've added a comment above to describe what they're doing -- if that's not enough to address your concern I'll reconsider expanding the variables. Let me know!

Reply:
I think this looks good!